sources/packages/api/src/route-controllers/designation-agreement/models/designation-agreement.model.ts

```diff
@@ -40,3 +47,23 @@ export interface GetDesignationAgreementsDto {
   startDate?: string;
   endDate?: string;
 }
+
+export interface PendingDesignationDto extends GetDesignationAgreementsDto {
+  institutionName: string;
+}
+
+export interface UpdateDesignationLocationDto {
+  designationLocationId?: number;
+  locationId: number;
+  requested: boolean;
+  approved: boolean;
+}
+
+export interface UpdateDesignationDto {
```

Comment:
During the update, all nullable properties in this object are actually mandatory, doesn't?

Reply:
the nullable properties are not used while declining designation. Comments updated as discussed.